### DISCUSSION TITLE
Mqtt

### DISCUSSION THREAD

**Discussion by stevoh6** [serkri/SmartEVSE-3#124] [Ideas]
HI, since Im new user and Im using home assistant as rest of us, it is possible to add MQTT meter support? It will be more reliable than API, it will not overload HA server ( since API updates needs to be send from HA to EVSE every 5sec. ), it can open new posibilities like instant updates to HA etc.

**Comment by dingo35** [comment]
Someone made an MQTT version, and I understood they would be coming back with an atomic Pull Request, but it all went quiet after that.

**Reply by stevoh6** [reply]
You mean this one? https://github.com/arpiecodes/SmartEVSE-3/tree/mqtt ? :-)

**Comment by dingo35** [comment]
Yes

**Reply by stevoh6** [reply]
thx :-)

**Comment by umverteiler** [comment]
This would be a great addition and I was looking for this feature just this week.
I have a setup to test this (Eastron SDM630 in MQTT) - if needed.
